test/functional/tools/samples_tool_conf.xml

```diff
@@ -155,6 +155,7 @@
   <tool file="collection_mixed_param.xml" />
   <tool file="collection_two_paired.xml" />
   <tool file="collection_creates_pair.xml" />
+  <tool file="collection_creates_pair_format.xml" />
```

Comment:
I think this file is missing in this PR?

Reply:
Oop. Forgot to add it. Btw. is it a problem that the `collection_creates_pair*` tools have the same id? Just noticed that `./run_tests.sh -framework -id collection_creates_pair` runs all of them.

Reply:
They don't seem to have the same id:

```
$ head -n 1 test/functional/tools/collection_creates_pair*
==> test/functional/tools/collection_creates_pair_from_type.xml <==
<tool id="collection_creates_pair_from_type" name="collection_creates_pair_from_type" version="0.1.0">

==> test/functional/tools/collection_creates_pair_from_work_dir.xml <==
<tool id="collection_creates_pair_from_work_dir" name="collection_creates_pair_from_work_dir" version="1.0.0">

==> test/functional/tools/collection_creates_pair.xml <==
<tool id="collection_creates_pair" name="collection_creates_pair" version="0.1.0">

==> test/functional/tools/collection_creates_pair_y.yml <==
id: collection_creates_pair_y
```

Reply:
True. Then it seems that `run_tests.sh` tests all tools with an id that has a prefix specified by `-id`